packages/suite-native-atoms/src/Select/Select.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { ReactNode, useState } from 'react';
+
+import { CryptoIcon, CryptoIconName, FlagIcon, FlagIconName } from '@trezor/icons';
+
+import { isCryptoIconType, isFlagIconType } from './selectTypes';
+
+import { BottomModal } from '../Modal/BottomModal';
+import { SelectTrigger } from './SelectTrigger';
+import { SelectItem } from './SelectItem';
+
+export type SelectItem = {
+    label: string;
+    value: string;
+    iconName?: FlagIconName | CryptoIconName;
+};
+
+type SelectProps = {
```

Comment:
I looked into the API of mui as well, but since I will need the icon name (difference between our component and theirs is that we display icons in select label / value as well) and label also. So it's either sending the whole object or spread the object which will result in nearly identical result imho. So I would opt out to just go with this approach maybe? We can have a call if I'm missing some other reasons to spread the object maybe.

Reply:
`selectedItem.value` is unique identifier of that item, aka ID so that's only info you need, so you never need to pass whole `selectedItem` except prop `items`. Let's have call later if you still have trouble to understand (reach me on Slack).

Reply:
[b54e127](https://github.com/trezor/trezor-suite/pull/5506/commits/b54e12792e1a024ee488adf91ca238da4f65e391)